projects/igniteui-angular/src/lib/slider/thumb/thumb-slider.component.ts

```diff
@@ -57,13 +57,13 @@ export class IgxSliderThumbComponent implements OnInit, OnDestroy {
     public deactiveState: boolean;
 
     @Output()
-    public onThumbValueChange = new EventEmitter<number>();
+    public thumbValueChange = new EventEmitter<number>();
```

Comment:
Those three emitter (thumbValueChange, thumbChange, hoverChange) are internal, so we do not have to have migrations for them.
Also, we can comment them as @hidden.